src/app/components/welcome-page/welcome-page.component.html

```diff
@@ -10,12 +8,12 @@ <h1 class="welcomeSign">Welcome to <br><strong>Iced Reddit</strong></h1>
         <div class="sidePanel">
             <div class="loginContainer">
                 <h2>To enter the application, please log in.</h2>
-                <button class="loginBtn" type="submit">Log in</button>
+                <button class="loginBtn" type="submit" (click)="loginClick();">Log in</button>
```

Comment:
As mentioned a while ago in a PR, event-based actions are often named differently. Without this context, think about:

- What triggers on `loginClick()` function call/happens after the call
- What triggers on `onLoginClick()` function call / happens after the call

As for me, for the second one, I can say for sure that `onLoginClick()` is called by a button click (because of the `on` prefix). What happens is that based on previously mentioned info, user is quite likely logged into the application.

For the `loginClick()`, it is not as clear - it could be that we are triggering "login button click" from code and you may want to investigate the code further and spend some more time on that (whereas in the first one you are pretty sure what's happening).

Let me know what do you think about this interpretation and if you feel it is flawed. See if you can update the naming.

Reply:
AAAAA. For sure onLoginClick() is far better. I will change that. Thanks.